de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5462240_sys_gh1513-translating_en_us.sql

```diff
@@ -0,0 +1,80 @@
+update ad_field_trl set name = 'Delivery method' ,istranslated = 'Y' where ad_field_id = 2877 and ad_language = 'en_US';
```

Comment:
hmmm... i think we can integrate this PR but i find it fragile (easy to trash your work) because:
* you updated the trls on AD_Field level
* i assume the AD_Field.IsCentrallyMaintained=Y which means: if somebody is changing the AD_Element trl, the changes will be propagated down to the AD_Field, so your trl will be overridden => see https://github.com/metasfresh/metasfresh/issues/1044

I would suggest:
* unset IsCentrallyMaintained flag
* or change the translation on AD_Element level if the translation is the same in all windows (sales/purchase)